src/patternfly/components/ToggleGroup/examples/ToggleGroup.md

```diff
@@ -0,0 +1,157 @@
+  {{#> toggle-group-button}}
+    Option 2
+  {{/toggle-group-button}}
+  {{> divider divider--type="div" divider--modifier="pf-m-vertical"}}
+  {{#> toggle-group-button toggle-group-button--attribute="disabled"}}
+    Option 3
+  {{/toggle-group-button}}
+{{/toggle-group}}
+```
+
+```hbs title=With-icon
+{{#> toggle-group}}
+  {{#> toggle-group-button toggle-group-button--attribute='aria-label="Copy button"'}}
+    <i class="fas fa-copy" aria-hidden="true"/>
+  {{/toggle-group-button}}
+  {{> divider divider--type="div" divider--modifier="pf-m-vertical"}}
+  {{#> toggle-group-button toggle-group-button--attribute='aria-label="Undo button"'}}
+    <i class="fas fa-undo" aria-hidden="true"></i>
+  {{/toggle-group-button}}
+  {{> divider divider--type="div" divider--modifier="pf-m-vertical"}}
+  {{#> toggle-group-button toggle-group-button--attribute='aria-label="Share button"'}}
+    <i class="fas fa-share-square" aria-hidden="true"></i>
+  {{/toggle-group-button}}
+{{/toggle-group}}
+
+<br/>
+
+{{#> toggle-group}}
+  {{#> toggle-group-button toggle-group-button--modifier="pf-m-selected" toggle-group-button--attribute='aria-label="Copy button"'}}
+    <i class="fas fa-copy" aria-hidden="true"/>
+  {{/toggle-group-button}}
+  {{> divider divider--type="div" divider--modifier="pf-m-vertical"}}
+  {{#> toggle-group-button toggle-group-button--attribute='aria-label="Undo button"'}}
+    <i class="fas fa-undo" aria-hidden="true"></i>
+  {{/toggle-group-button}}
+  {{> divider divider--type="div" divider--modifier="pf-m-vertical"}}
+  {{#> toggle-group-button toggle-group-button--attribute='aria-label="Share button"'}}
+    <i class="fas fa-share-square" aria-hidden="true"></i>
+  {{/toggle-group-button}}
+{{/toggle-group}}
+
+<br/>
+
+{{#> toggle-group}}
+  {{#> toggle-group-button toggle-group-button--attribute='aria-label="Copy button"'}}
+    <i class="fas fa-copy" aria-hidden="true"/>
+  {{/toggle-group-button}}
+  {{> divider divider--type="div" divider--modifier="pf-m-vertical"}}
+  {{#> toggle-group-button toggle-group-button--attribute='aria-label="Undo button"'}}
+    <i class="fas fa-undo" aria-hidden="true"></i>
+  {{/toggle-group-button}}
+  {{> divider divider--type="div" divider--modifier="pf-m-vertical"}}
+  {{#> toggle-group-button toggle-group-button--attribute='aria-label="Share button" disabled'}}
+    <i class="fas fa-share-square" aria-hidden="true"></i>
+  {{/toggle-group-button}}
+{{/toggle-group}}
+```
+
+```hbs title=Light
+{{#> toggle-group}}
+  {{#> toggle-group-button toggle-group-button--modifier="pf-m-light"}}
+    Option 1
+  {{/toggle-group-button}}
+  {{> divider divider--type="div" divider--modifier="pf-m-vertical"}}
+  {{#> toggle-group-button toggle-group-button--modifier="pf-m-light"}}
+    Option 2
+  {{/toggle-group-button}}
+  {{> divider divider--type="div" divider--modifier="pf-m-vertical"}}
+  {{#> toggle-group-button toggle-group-button--modifier="pf-m-light pf-m-selected"}}
+    Option 3
```

Comment:
I'm assuming you could also put an icon in a button? Since the text and icons have no BEM element, that will require updates to handle properly if we want to manage that spacing.